.env.example

```
@@ -2,7 +2,8 @@ DISCORD_TOKEN=YOUR_DICORD_TOKEN
 GUILD_ID=SERVER_GUID
 SUPPORT_CHANNEL=ID_SUPPORT_CHANNEL
 GITHUB_TOKEN=YOUR_GITHUB_TOKEN
-GITHUB_REPO=organization/repo
+REPO_TARKOVTRACKER=organization/repo
+REPO_DATA_REPORT=tarkovtracker-org/tarkov-data/overlay
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The REPO_DATA_REPORT key should go before the REPO_TARKOVTRACKER key

(UnorderedKey)